frontend-vue/src/pages/domain-details/DomainDetails.vue

```diff
@@ -261,6 +265,16 @@ function onEditCloseClick() {
   editMode.value = false;
 }
 
+watch(
```

Comment:
Personally I strive to avoid using watch() too often as it makes it hard to follow the data flow. Can we use a computed property instead?

const canCreateSubdomain = computed(() => { /* recalculate the relative level to parent /*});

Reply:
I agree with you, in this case to ensure the render/logic is updated first and consistently the `watch` property here works as expected.
Alternatively we could implement a Middleware for route changes altough this could lead to overengineering this logic.